opm/material/fluidsystems/BlackOilFluidSystem.hpp

```diff
@@ -657,6 +691,26 @@ class BlackOil : public BaseFluidSystem<Scalar, BlackOil<Scalar> >
         referenceDensity_.resize(numRegions);
     }
 
+    template <class LhsEval, class FluidState>
+    static LhsEval getRs_(const FluidState& fluidState, unsigned regionIdx)
+    {
+        typedef Opm::MathToolbox<typename FluidState::Scalar> FsToolbox;
+
+        const auto& XoG =
+            FsToolbox::template toLhs<LhsEval>(fluidState.massFraction(oilPhaseIdx, gasCompIdx));
+        return XoGToRs(XoG, regionIdx);
```

Comment:
Well, would `gasMassFractionInOilToGasDissolutionFactor()` be better? To keep consistency, I'd like to use a capital `X` for mass fractions. (lower-case `x` means mole fraction.)

In any case, this is a blackoil specific method which is not included in the fluid system API.